toolbox

```diff
@@ -1259,7 +1267,12 @@ usage()
 
 arguments=$(save_positional_parameters "$@")
 
-release_default=$(get_host_version_id)
+host_id=$(get_host_id)
+if [ "$host_id" = "fedora" ]; then
```

Comment:
Nitpick: would be nice to redirect the standard error to file descriptor 3 for `--verbose` output.

Reply:
I took the liberty to tweak it as above.